core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -417,6 +417,13 @@ class ZkUtils(val zkClient: ZkClient,
     topicDirs.consumerOwnerDir + "/" + partition
   }
 
+  def controllerZkData(brokerId: Int, timestamp: Long): String = {
+    Json.encode(Map("version" -> 1, "brokerid" -> brokerId, "timestamp" -> timestamp.toString))
+  }
+
+  def preferredReplicaLeaderElectionZkData(partitions: scala.collection.Set[TopicAndPartition]): String = {
```

Comment:
Yes, I agree that it should have been like that in the first place.

`ZkUtils` is not considered public API, so it's technically OK to make changes like that (and we have done them in previous releases). However, I'm also fine with leaving the existing methods where they are.